src/Components/Footer/Footer.css

```diff
@@ -0,0 +1,18 @@
+.footer {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  height: 80px; /*sets the height of the footer, can adjust if we want it larger/smaller */
+  background-color: #F7F7F9;
+  position: absolute;
+  bottom: 0;
+  width: 100%;
+}
+
+.footer p {
+  margin: 0;
+  font-size: 16px;
```

Comment:
Could you please change font size to rem instead of px? Font size should always be relative as px is not accessible (eg 16px is big on a phone screen but tiny on an ultra wide)